docker-compose.yml

```diff
@@ -150,16 +150,18 @@ services:
       start_period: 40s
 
   simulator:
-    image: mojaloop/simulator:latest
+    image: mojaloop/simulator-perf:m9.2.1-perf4.1.2
```

Comment:
Please use an official Simulator release. I.e. if we need, we will have to also migrate perf changes into master branch.